fixture_rails_root/db/migrate/20190110100325_add_commercial_entity_to_plutus_accounts.rb

```diff
@@ -0,0 +1,7 @@
+class AddCommercialEntityToPlutusAccounts < ActiveRecord::Migration[4.2]
+  def change
+    add_column :plutus_accounts, :commercial_entity_id, :integer
+    add_column :plutus_accounts, :commercial_entity_type, :string
+    add_index :plutus_accounts, [:commercial_entity_type, :commercial_entity_id], :name => "index_accounts_on_commercial_entity"
```

Comment:
index name could conflict. maybe:

```suggestion
    add_index :plutus_accounts, [:commercial_entity_type, :commercial_entity_id], :name => "index_plutus_accounts_on_commercial_entity"
```